src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -68,6 +69,16 @@ private void OnParseCommand(string value)
             }
         }
 
+        /// <summary>
+        /// Read file from path and print its content in base64
+        /// </summary>
+        [ParseFunction("File path to content base64")]
+        private string? FilePathToContentBase64(string path)
+        {
+            if (!File.Exists(path)) return null;
```

Comment:
Make it allow only `nef` file extensions.

Reply:
Done